.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
- [ ] I cannot see any .gitignore file in your project. Kindly add that file to follow the requirements :blush: